stable/artifactory/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: artifactory
 home: https://www.jfrog.com/artifactory/
-version: 5.3.2
-appVersion: 5.3.2
+version: 5.4.1
```

Comment:
Chart version and app version are two different things. They don't have to be aligned.